features/nanostack/FEATURE_NANOSTACK/nanostack_entropy_check.cpp

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2015 ARM Limited. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ * Licensed under the Apache License, Version 2.0 (the License); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "mbedtls/platform.h"
```

Comment:
I think this should really be `#include "mbedtls/config.h"` which is the file that contains all the library compile time switches. `mbedtls/platform.h` is completely different and defines things like `calloc`, `printf`, timing functions, etc.